src/web/modules/transactions/update/update.http.ts

```diff
@@ -79,7 +80,7 @@ const validateAndAddDefaults = async function validateAndAddDefaults(csv: string
     parseString<TransactionRow, TransactionRow>(csv, { headers: true })
       .transform((row: TransactionRow) => {
         if (!row.event_date) {
-          row.event_date = moment().utc().format()
+          row.event_date = `${moment().utc().format('YYYY-MM-DDTHH:mm:ss.SSSSSS')}Z`
```

Comment:
do we need to improve the validation for user provided dates? does ledger expect a very specific format?

Reply:
yes this would be a good idea ^ potentially Toolbox could also parse the date in a flexible way and then reformat it for exactly what Ledger needs. 

I propose we do this in a separate PR